lib/inc/drogon/HttpAppFramework.h

```diff
@@ -175,6 +175,16 @@ class DROGON_EXPORT HttpAppFramework : public trantor::NonCopyable
                                       const HttpRequestPtr &req)>
             &&resp_generator) = 0;
 
+    virtual HttpAppFramework &setCustomErrorHandler(
```

Comment:
The virtual keyword is not needed here.

Reply:
Doesn't that break backward compatibility?